tests/verify_heap.cpp

```diff
@@ -664,7 +660,11 @@ auto run_heap_verification(
     const unsigned threads,
     const bool machine_readable) -> bool
 {
-    cudaSetDeviceFlags(cudaDeviceMapHost);
```

Comment:
Is that still needed? I removed it because it seemed to not make a difference.

Reply:
Yes remove it  I do not think it is required

Reply:
Great!